Source/Turbo Navigator/TurboNavigator.swift

```diff
@@ -105,16 +105,27 @@ public class TurboNavigator {
             return
         }
     }
-
-    let session: Session
-    let modalSession: Session
-
+    
+    public func appDidBecomeActive() {
+        appInBackground = false
+        inspectAllSession()
+    }
+    
+    public func appDidEnterBackground() {
+        appInBackground = true
+    }
+    
+    var session: Session
+    var modalSession: Session
+    
     /// Modifies a UINavigationController according to visit proposals.
     lazy var hierarchyController = TurboNavigationHierarchyController(delegate: self)
-
+    
```

Comment:
I'm (almost) fine with the whitespace-only lines that already exist in the repo but let's not add any new ones.

Reply:
I removed the line altogether.